wp-includes/components/class-wp-service-worker-caching-routes.php

```diff
@@ -136,4 +136,84 @@ public function register( $route, $args = array() ) {
 	public function get_all() {
 		return $this->routes;
 	}
+
+
+	/**
+	 * Prepare caching strategy args for export to JS.
+	 *
+	 * @since 0.2
+	 *
+	 * @param array $strategy_args Strategy args.
+	 * @return string JS IIFE which returns object for passing to registerRoute.
+	 */
+	public static function prepare_strategy_args_for_js_export( $strategy_args ) {
```

Comment:
Ah! Having this here makes a lot more sense to me, and then having the method being static for outside access appears somewhat okay too - I'm sure we can find a better architecture for this, but I'm okay with proceeding as is for now.